### PR TITLE
Import binary and hex

### PR DESCRIPTION
**Pull request by wkwiatek**

### CODE REVIEW

File: src/components/ProgramLoader/BinaryFileUpload.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  };
+
+  return (
+    <div className="block">
+      <p className="mt-10 mb-3">or upload program as a binary file</p>
+      <Input
+        className="mt-3 mr-3"
+        id="test-file"
+        type="file"
+        // accept="application/json"
```
Comment (tomusdrw) [inline src/components/ProgramLoader/BinaryFileUpload.tsx:52]:
```suggestion
        accept=".json,.bin"
```

**Review by tomusdrw — APPROVED** [review]

File: src/components/ProgramLoader/BinaryFileUpload.tsx (hunk 1 above)
Comment (wkwiatek) [reply src/components/ProgramLoader/BinaryFileUpload.tsx:52]: I think `json` is no longer valid here as this is the case just for the binary file. 

If there's no other specific file extension for binary files containing the program I'd go with just `.bin`

Comment (tomusdrw) [reply src/components/ProgramLoader/BinaryFileUpload.tsx:52]: Sounds good, I thought that the component might be shared with the JSON test files upload, hence the suggestion, but if it's just here, then `.bin` is fine.

Comment (wkwiatek) [reply src/components/ProgramLoader/BinaryFileUpload.tsx:52]: It turned out that the only common part is the looks and we may also change the look and feel here as well (independently to the test file loaded) so was the idea behind. 

Changing then.

**Pull request merged**